### DISCUSSION TITLE
Datenpunktwert in die Beschreibung

### DISCUSSION THREAD

**Discussion by tschortschde** [mcuiobroker/docuJarvis#25] [Q&A]
Hallo,
ich möchte gerne den Wert des DP in die Beschreibung vor "Fenster offen bringen" bzw. wenn alle Fenster geschlossen sind den Text "Alle Fenster geschlossen" in der Beschreibung anzeigen.
Ist das möglich und wie?
![Screenshot 2023-03-24 172900](https://user-images.githubusercontent.com/11274233/227585736-4f80b158-d948-4367-b516-7857fb84674c.jpg)

**Comment by mcuiobroker** [comment]
Versuch mal im Label
```
{"default":"{val} Fenster offen","0":"Alle Fenster geschlossen"}
```
evtl. erst ab v3.1.4-beta.6??
